raiden/api/python.py

```diff
@@ -815,3 +817,35 @@ def get_blockchain_events_channel(
             ))
         returned_events.sort(key=lambda evt: evt.get('block_number'), reverse=True)
         return returned_events
+
+    def create_monitoring_request(
+            self,
+            payment_identifier: typing.PaymentID,
+            reward_amount: typing.TokenAmount,
+    ) -> RequestMonitoring:
```

Comment:
the type is wrong. It seems the function can also return `None` so it should be `Optional[RequestMonitoring]`